### PR TITLE
Update Set-ADReplicationSite.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected KCC from   Kerberos consistency checker to  Knowledge Consistency Checker.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2126

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 1]
```diff
@@ -602,7 +602,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyCleanupEnabled
-When enabled,  this optional parameter prevents the Kerberos consistency checker (KCC) from removing connection objects that it does not need.
+When enabled,  this optional parameter prevents the Knowledge consistency checker (KCC) from removing connection objects that it does not need.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:605]:
```suggestion
When enabled,  this optional parameter prevents the Knowledge Consistency Checker (KCC) from removing connection objects that it does not need.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 2]
```diff
@@ -619,7 +619,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyDetectStaleEnabled
-This parameter option prevents the Kerberos consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
+This parameter option prevents the Knowledge consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:622]:
```suggestion
This parameter option prevents the Knowledge Consistency Checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 3]
```diff
@@ -635,7 +635,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyMinimumHopsEnabled
-When enabled, this parameter prevents the Kerberos consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
+When enabled, this parameter prevents the Knowledge consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:638]:
```suggestion
When enabled, this parameter prevents the Knowledge Consistency Checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 4]
```diff
@@ -729,7 +729,7 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2003KCCBehaviorEnabled
-Implements Kerberos consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
+Implements Knowledge consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:732]:
```suggestion
Implements Knowledge Consistency Checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 5]
```diff
@@ -760,7 +760,7 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2003KCCSiteLinkBridgingEnabled
-When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Kerberos consistency checker (KCC) control of the ability to enable or disable site link bridging.
+When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge consistency checker (KCC) control of the ability to enable or disable site link bridging.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:763]:
```suggestion
When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge Consistency Checker (KCC) control with the ability to enable or disable site link bridging.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**